### PR TITLE
Update README and ATL display to reflect ATL grammar

### PR DESCRIPTION
**Pull request by NicEastvillage**
Made ATL display and ATL grammar consistent by replacing the special characters with `<<`, `>>`, `[[`, `]]`, `F`, `G`, `X`.

And updated README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # atl-checker
 
 This application can do model checking of alternating-time temporal logic (ATL) on concurrent game structures (CGSs).
-It uses a on-the-fly evaluation method as opposed to calculating the fixed point like PRISM do.
-This lazy method avoids having to generate the whole graph, and is therefore faster in most cases (has yet to be proved).
+It uses an on-the-fly evaluation method as opposed to calculating the fixed point like PRISM do.
```
Comment (MathiasMehl) [inline README.md:4]:
```suggestion
It uses an on-the-fly evaluation method as opposed to calculating the fixed point like PRISM does.
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+This lazy method avoids having to generate the whole graph, and is therefore faster in most cases.
```
Comment (MathiasMehl) [inline README.md:5]:
```suggestion
This lazy evaluation method avoids generating the whole graph, and is therefore faster in most cases.
```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
 The ATL-checker also uses a custom language to describe CGSs in a lazy manner. We call it LCGS and
 it's syntax is inspired by PRISM-lang, however, the concepts of syncronization and modules are very different.
```
Comment (MathiasMehl) [inline README.md:8]:
```suggestion
its syntax is inspired by PRISM-lang, however, the concepts of synchronization and modules are very different.  
```

**Review by MathiasMehl — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
 # atl-checker
 
 This application can do model checking of alternating-time temporal logic (ATL) on concurrent game structures (CGSs).
-It uses a on-the-fly evaluation method as opposed to calculating the fixed point like PRISM do.
-This lazy method avoids having to generate the whole graph, and is therefore faster in most cases (has yet to be proved).
+It uses an on-the-fly evaluation method as opposed to calculating the fixed point like PRISM does.
+With this lazy method we avoid generating the whole graph, and it is therefore faster in most cases.
```
Comment (cogitantium) [inline README.md:5]:
```suggestion
With this lazy method we avoid generating the whole graph, and it is therefore an order of magnitude faster in most cases.
```

We can link to SciHub/arXiv later on for this statement

**Review by cogitantium — CHANGES_REQUESTED** [review]

**Review by cogitantium — APPROVED** [review]

**Pull request merged**